website/public/css/index.styl

```diff
@@ -110,6 +110,12 @@ html, body
 -------------------------------------------------- */
 //move
 
+.modal-subheading {
+  margin: 12px 0 8px;
+  font-size: 24px;
+  font-weight: bold;
+}
```

Comment:
I feel like there is a bootstrap class we could leverage instead of creating a new class. Maybe a combination of a strong element nested in a [lead class](http://getbootstrap.com/css/#lead-body-copy)?

```
p.lead: strong {{::removeMemberData.member.profile.name}}
```